CONTRIBUTING.md

```diff
@@ -295,6 +295,10 @@ Run selected tests:
 python3 -m pytest path/to/file/or/directory/to/test
 ```
 
+Run the cpu only tests with `pytest -m 'not needs_gpu' ./tensorflow_addons`.
+Run the gpu only tests with `pytest -m needs_gpu ./tensorflow_addons`.
```

Comment:
```suggestion
Run the gpu only tests with `pytest -m needs_gpu ./tensorflow_addons`.
Run the cpu only tests with `pytest -m 'not needs_gpu' ./tensorflow_addons`.
```

Very minor, but think it's a bit more intuive in the doc to read the positive and then the negative afterward. Similar to how it's done on pytest markers docs. This pytest marker methodology is also not my usage pattern but it's very useful